CMakeLists.txt

```diff
@@ -18,10 +18,30 @@ endif()
 
 enable_testing()
 
-# Leatherman setup
-list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/vendor/leatherman/cmake")
+include(FeatureSummary)
+
+SET(LEATHERMAN_COMPONENTS locale catch nowide logging util file_util dynamic_library execution ruby rapidjson)
```

Comment:
Project doesn't need all of these. If we're just including them as an example, add a comment to reflect that and mention which are actually used by the project.